### ISSUE TITLE
Cannot launch miss variables

### ISSUE THREAD

**Issue by Camiyo** [axel3rd/mpg-coach-bot#118]
Hey there,

I'm not very comfortable with java application, I put my credentials in mpg.properties and I tried to launch your sh script :

```
$ ./mpg-coach-bot.sh 
./mpg-coach-bot.sh: ligne 3: ${project.artifactId}-${project.version}.jar : mauvaise substitution
```

I couldn't found any .jar anywhere and I don't know what these variables should contain :+1: 
 ${project.artifactId}-${project.version}.

Thanks for helping, I probably miss something.

**Comment by axel3rd** [comment]
Helllo @Camiyo, 

You seems have download the "repository release" from https://github.com/axel3rd/mpg-coach-bot/releases (containing the code, not the produced *jar*).

Please use the **Download** button in top of README description (https://github.com/axel3rd/mpg-coach-bot, below the listed files). 
For last current development version, this is https://bintray.com/axel3rd/generic-dev/mpg-coach-bot/1.5-SNAP#files

**Comment by Camiyo** [comment]
Ahah ok my bad, it's working !
Thanks,

**Comment by Camiyo** [comment]
Just one last question :) 
We can see which player got an injuries, but not a player who got a red card ?

Like my guy there : 
| D | Boly Willy             |  5,50 | 21 |

Thanks !

**Comment by axel3rd** [comment]
> We can see which player got an injuries, but not a player who got a red card ?

In theory, yes: But Boly Willy seems not have red card in http://www.equipeactu.fr/blessures-et-suspensions/fodbold/angleterre/championship (third part website for injuries/suspended).

**Comment by Camiyo** [comment]
Arf ok, thank you !

**Comment by Camiyo** [comment]
Hey, we look at the
http://www.equipeactu.fr/blessures-et-suspensions/fodbold/angleterre/championship
but Boly is in
http://www.equipeactu.fr/blessures-et-suspensions/fodbold/angleterre/
*premier-league*
<http://www.equipeactu.fr/blessures-et-suspensions/fodbold/angleterre/championship>

**Comment by axel3rd** [comment]
I didn't found "Bolly" in this link :confounded:

**Comment by Camiyo** [comment]
=> Team Wolverhampton

**Comment by axel3rd** [comment]
> curl http://www.equipeactu.fr/blessures-et-suspensions/fodbold/angleterre/premier-league |grep Boly
> Team Wolverhampton

Heu ... very very good catch. For Premiere League, the link for injured/suspended players is bad 😢 => #119

**Comment by axel3rd** [comment]
@Camiyo Thanks !!!

**Comment by axel3rd** [comment]
> Heu ... very very good catch. For Premiere League, the link for injured/suspended players is bad 😢 => #119

@Camiyo : Premiere League out players (injured/suspended) fixed in [mpg-coach-bot-1.5-SNAPSHOT.zip](https://bintray.com/axel3rd/generic-dev/download_file?file_path=mpg-coach-bot-1.5-SNAPSHOT.zip)
